types/svg-baker-runtime/src/browser-sprite.d.ts

```diff
@@ -0,0 +1,46 @@
+import Sprite from './sprite';
+import BrowserSpriteSymbol from './browser-symbol';
+
+import SpriteConfig from './sprite.config';
+import BrowserSpriteConfig from './browser-sprite.config';
+
+export default class BrowserSprite extends Sprite {
+    constructor(cfg?: SpriteConfig & BrowserSpriteConfig);
+
+    config: SpriteConfig & BrowserSpriteConfig;
+    node: Element | null;
```

Comment:
this this one supposed to be public? It's access via 'root' getters or queries for state with 'isMounted', but is this used directly somewhere? (breaking encapsulation intent)

Reply:
In my opinion, these properties/methods starts with underscore should be private and others should be public or protected. Like these samples: 
* `_autoConfigure`: https://github.com/JetBrains/svg-mixer/blob/v1/packages/svg-baker-runtime/src/browser-sprite.js#L93
* `https://github.com/JetBrains/svg-mixer/blob/v1/packages/svg-baker-runtime/src/browser-sprite.js#L116` https://github.com/JetBrains/svg-mixer/blob/v1/packages/svg-baker-runtime/src/browser-sprite.js#L116
* `_emitter` https://github.com/JetBrains/svg-mixer/blob/v1/packages/svg-baker-runtime/src/browser-sprite.js#L32

And I can see some `@private` jsdoc markers in the source code, like [this search result in sourcegraph](
https://sourcegraph.com/search?q=context:global+repo:%5Egithub%5C.com/JetBrains/svg-mixer%24%40v1+%40private&patternType=literal)

![image](https://user-images.githubusercontent.com/6828924/160547050-1bbd35f5-5079-4bc2-9479-06b6bb08bc27.png)

So I would like to change this private marked properties/methods to private in d.ts, and others would following the reviews and the source codes themselves.

Reply:
OK, so I would say `destron` is protected (or @internal to lib), there is no mention of any kind of destroy calls on docs site:
https://github.com/JetBrains/svg-mixer/tree/master/packages/svg-mixer

My point here is only, if the declarations files are not autogenerated, there is no need to provide private/internal symbols, the module is to be consumed by client code.